variants/backend-base/example.env.tt

```diff
@@ -26,3 +20,13 @@ PORT=3000
 # SENTRY_DSN=http://public@example.com/project-id
 SENTRY_CSP_HEADER_REPORT_ENDPOINT=https://SOMECODE.ingest.sentry.io/api/SOMENUMS/security/?sentry_key=SOMETHING
 SENTRY_ENV=development
+
+# NEVER just copy these secrets from development env to a production env.  Run:
+#
+#     bin/rails db:encryption:init
+#
+# to create new versions of these secrets for each deployed environment (e.g.
+# staging, production)
+ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY="PLACEHOLDER_PRIMARY_KEY"
+ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY="PLACEHOLDER_DETERMINISTIC_KEY"
+ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT="PLACEHOLDER_KEY_DERIVATION_SALT"
```

Comment:
All [`bin/rails db:encryption:init` does is call `SecureRandom.alphanumeric(32)` three times](https://github.com/rails/rails/blob/74e24aef41da2482fa896684a7b4bccc6afc9b3b/activerecord/lib/active_record/railties/databases.rake#L533-L545) - I think that would be the simpler thing to do, and would match how we generate `RAILS_SECRET_KEY_BASE`:

```suggestion
ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY=<%= SecureRandom.alphanumeric(32) %>
ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY=<%= SecureRandom.alphanumeric(32) %>
ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT=<%= SecureRandom.alphanumeric(32) %>
```

Reply:
i agree

Reply:
Will change.